catchup/peerSelector.go

```diff
@@ -185,7 +185,6 @@ func (ps *peerSelector) PeerDownloadDurationToRank(peer network.Peer, blockDownl
 	default: // i.e. peerRankInitialFourthPriority
 		return downloadDurationToRank(blockDownloadDuration, lowBlockDownloadThreshold, highBlockDownloadThreshold, peerRank3LowBlockTime, peerRank3HighBlockTime)
 
-		
```

Comment:
please undo this change. you didn't really meant to make it, right ?

Reply:
changes like these were applied after I ran make fmt. should I revert them?

Reply:
yes. In general, you want to ensure that you apply your changes only on the files that you've modified.
( it's ok to make general codebase "fixes". But I'd rather these to be placed on a separate PR )